pom.xml

```diff
@@ -37,7 +37,7 @@
 
         <commons-io.version>2.4</commons-io.version>
         <apache-commons-lang3.version>3.5</apache-commons-lang3.version>
-        <java-dogstatsd-client.version>2.9.0</java-dogstatsd-client.version>
+        <java-dogstatsd-client.version>2.10.3</java-dogstatsd-client.version>
```

Comment:
This new version adds telemetry metrics (https://github.com/DataDog/java-dogstatsd-client/pull/97). I think this telemetry could mislead support and our end users into thinking that an actual java app is reporting dogstatsd metrics to the Agent (not just JMXFetch). The fact that JMXFetch uses dogstatsd is an implementation detail our users shouldn't care about.

So let's disable this telemetry on the client.